### ISSUE TITLE
Unable to add new integration

### ISSUE THREAD

**Issue by trek-boldly-go** [natekspencer/hacs-vivint#156]
### The problem

Running 2024.11.1 of the integration
Tested with HA 2024.11.0 and 2024.11.1, same error on both versions

HA is running in a docker container. This integration has worked for a long time for me in the past, but after a necessary upgrade of HA, this integration broke (see #151 ) and I have been unable to set it up since. 

The sign in popup merely says "Unexpected error", but this is the only error I see in the logs:
```
This error originated from a custom integration.

Logger: custom_components.vivint.config_flow
Source: custom_components/vivint/config_flow.py:119
integration: Vivint (documentation, issues)
First occurred: November 17, 2024 at 4:42:58 PM (2 occurrences)
Last logged: 4:00:23 PM

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/vivint/config_flow.py", line 119, in async_vivint_login
    await self._hub.login(load_devices=True)
  File "/config/custom_components/vivint/hub.py", line 96, in login
    self.account = Account(
                   ^^^^^^^^
TypeError: Account.__init__() got an unexpected keyword argument 'refresh_token'
```

I know a little bit of python, but I really can't tell what the issue is here. Maybe HA didn't download the newest version of vivintpy? If it didn't, how would I force it? I have used the delete function in HACS, rebooted, installed the integration again, then rebooted and tried to sign in, but I still got the same error.

### What version of Home Assistant Core are you running?

2024.11.0

**Comment by trek-boldly-go** [comment]
I went digging through the file system and found multiple installations of vivintpy. There was one at `/config/deps/vivintpy` (remember this is a container so the path may be different on HA OS) which was correctly at 2024.2.1, but there was also one located at `/config/deps/lib/python3.12/site-packages/vivintpy` which was from 2023 (I don't recall the specific version). 

On a hunch, I replaced that second install with a copy of the first one, and rebooted HA. That seemed to fix the issue. I was able to sign in and discover all my devices again. I don't know why HA is using the copy of vivintpy that it is and I don't know why that copy wasn't updated. Either way, I just hope this doesn't become a regular problem any time vivintpy gets an upgrade. 

Thanks!
